### ISSUE TITLE
JS error - wp is not defined

### ISSUE THREAD

**Issue by benitolopez** [thomasgriffin/TGM-Plugin-Activation#708]
Hi,

Every time I update a plugin through TGMPA I receive a JS error at the end of the process:

`Uncaught TypeError: Cannot read property 'decrementCount' of undefined`

The code fails in this script:

```
<script type="text/javascript">
	(function( wp ) {
		if ( wp && wp.updates.decrementCount ) {
			wp.updates.decrementCount( "plugin" );
		}
	})( window.wp );
</script>
```

Any idea?

**Comment by jrfnl** [comment]
Thanks for reporting this. TGMPA does not have its own javascript, but uses some of the scripts from WordPress Core. Seems they have been making some changes over the last few versions causing notices without impeding functionality. This needs to be investigated, especially how to get this working notice free across all supported WP versions.

**Comment by benitolopez** [comment]
Yes, the "problem" it's inside the `decrement_update_count()` method which is called in the `after()` method of the `Plugin_Upgrader_Skin` class.

**Comment by jrfnl** [comment]
Side-note: this may be loosely related to #660 which also reports a notice coming from the WP core JS files.
